vent/coordinator/message.py

```diff
@@ -0,0 +1,89 @@
+from enum import Enum, auto
+
+
+class SensorValues:
+    def __init__(self, pip=None, peep=None, fio2=None, temp=None, humidity=None, vte=None, breaths_per_minute=None,
+                 inspiration_time_sec=None, timestamp=None):
+        self.pip = pip
+        self.peep = peep
+        self.fio2 = fio2
+        self.temp = temp
+        self.humidity = humidity
+        self.vte = vte
+        self.breaths_per_minute = breaths_per_minute
+        self.inspiration_time_sec = inspiration_time_sec
+        self.timestamp = timestamp
+
+
+class ControlSettingName(Enum):
+    PIP = auto()
+    PEEP = auto()
+    FIO2 = auto()
+    TEMP = auto()
+    HUMIDITY = auto()
+    VTE = auto()
+    BREATHS_PER_MINUTE = auto()
+    INSPIRATION_TIME_SEC = auto()
+
+
+class ControlSettings:
+    def __init__(self, name, value, min_value, max_value, timestamp):
+        """
+        TODO: if enum is hard to use, we may just use a predefined set, e.g. {'PIP', 'PEEP', ...}
+        :param name: enum belong to ControlSettingName
```

Comment:
not sure if you saw, but i have setup the docs to build w/ google-style docstrings 

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

these should still work, so totally fine, just wanted to make sure you were aware :)

Reply:
Oh thanks for this doc! Later I'll try to refine my documentation to fit this!